### PR TITLE
updates for 16

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: kitchen.dokken.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  name: dokken
+  product_name: chef
+  product_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
+  install_strategy: once
+
+platforms:
+  - name: centos-8
+    driver:
+      image: dokken/centos-8
+      pid_one_command: /usr/lib/systemd/systemd
```
Comment (ramereth) [inline kitchen.dokken.yml:20]: Please remove this.

**Comment by ramereth** [comment]
@TheREK3R you still need to resolve the conflicts.

**Comment by TheREK3R** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v5.4.0.  Have a nice day!

**Pull request merged**